src/Tasks/Engines/DoormanRunner.php

```diff
@@ -47,7 +48,7 @@ public function runQueue($queue)
 
         // split jobs out into multiple tasks...
 
-        $manager = new ProcessManager();
+        $manager = Injector::inst()->create('AsyncPHP\Doorman\Manager\ProcessManager');
```

Comment:
This is all good, but I'd personally prefer to see `->create(ProcessManager::class)`